api/auth/authproviders.py

```diff
@@ -66,7 +66,7 @@ def validate_code(self, code, **kwargs):
             'access_token': code,
             'uid': uid,
             'auth_type': self.auth_type,
-            'expires': datetime.datetime.utcnow() + datetime.timedelta(hours=1)
+            'expires': datetime.datetime.utcnow() + datetime.timedelta(days=14)
```

Comment:
I'll add a ticket to make this easier to understand/request that our JWT service gives us an expiry timestamp.